### ISSUE TITLE
Create single, cross-platform output script in Ruby

### ISSUE THREAD

**Issue by arthurattwell** [electricbookworks/electric-book#98]
At the moment, we have three output scripts, one for each major OS. This is only really the result of my shell-script-level skill set. A better solution is to have one script for all OSes. If should probably be written in Ruby, since users have to have Ruby running anyway for Jekyll etc.

**Comment by SteveBarnett** [comment]
I think it would make sense that [the Electric Book Manager](https://github.com/electricbookworks/electric-book-gui) would run this same script, for consistency of action and output (between running the EBW locally and via the EBM).

**Comment by arthurattwell** [comment]
@SteveBarnett Indeed. I've raised this in https://github.com/electricbookworks/electric-book-gui/issues/19.

**Comment by arthurattwell** [comment]
I'm having a go at a cross-platform output script in Node, with a view to being able to run it from an Electron app. That might give us a way to run an Electric Book project locally without using the command line.

I've gone a baby step towards a proof-of-concept Electron app in https://github.com/electricbookworks/electric-book-electron-poc, and initial work-in-progress on a Node script in the [`node-runner` branch here](https://github.com/electricbookworks/electric-book/compare/node-runner). All still pretty speculative, but intriguing.
